programs/streamflow-sdk/src/lib.rs

```diff
@@ -155,7 +241,31 @@ pub mod streamflow_sdk {
         Ok(())
     }
 
-    /// Anchor rpc handler used for CPI code generation
+    /// Create a Stream and skip some optional checks
+    ///
+    /// This method allows to create a stream and omit some of address checks on creation.
+    /// It is not recommended to use this method unless you need create a stream inside a contract and you don't have space for extra accounts.
+    ///
+    /// # Arguments
+    ///
+    /// * `ctx` - Accounts that will be used on Stream creation, `metadata` account shuold be initialized!
```

Comment:
Same comment as on the previous CTX. Also, metadata part can be used in the previous as well?